src/main.rs

```diff
@@ -118,6 +123,17 @@ fn main() {
             None => edit_snippet(),
         };
         let args = matches.value_of("args").unwrap_or("").to_string();
+        if matches.is_present("shorten") {
```

Comment:
@ethanhs would you have any better ideas how to structure the option handling? I don't find my solution very elegant but it works.

Reply:
It seems acceptable and in line with other option handling.